lua/tardis/parts/legacy/legacy_throttle.lua

```diff
@@ -15,12 +15,12 @@ if SERVER then
         if self.exterior:GetData("teleport")==true or self.exterior:GetData("vortex")==true then
             self.exterior:Mat()
         end
-        if self.interior:GetData("legacy-advanced",false)==true then return end
+        if TARDIS:GetSetting("csequences-enabled",false,self:GetCreator()) then return end
```

Comment:
This line probably shouldn't be here and should be handled by what I said before around handling this in the part system directly. Although it's a good point here actually because we need to handle the case of remat so it might be ok in this instance. Wasn't there a better .SequenceEnabled property you used on the other one you could use here? This call is a bit messy :(

Reply:
InSequence only gets set on the controls once the sequence is activated and they're a part of it. Maybe synchronise the setting with a data value?

Reply:
Maybe like self:GetSequencesEnabled or something that just calls that code yeah